packages/fuels-abigen-macro/tests/harness.rs

```diff
@@ -19,6 +19,13 @@ fn null_contract_id() -> String {
     String::from("0000000000000000000000000000000000000000000000000000000000000000")
 }
 
+async fn get_single_wallet() -> LocalWallet {
```

Comment:
I feel that this `single_wallet` should be a method for the `WalletConfig`

Reply:
More precisely, `single_wallet_config` should already exist in the `WalletConfig` (maybe create an `Enum` around it somewhere